src/pages/index.tsx

```diff
@@ -1,9 +1,16 @@
 import type { NextPage } from 'next'
 import Head from 'next/head'
-import { Box, Flex, Button } from "@chakra-ui/react";
+import { Box, Flex, Button, Textarea, Badge, Image } from '@chakra-ui/react'
 import { FaTwitter } from 'react-icons/fa'
+import { useState } from 'react'
 
 const Home: NextPage = () => {
+  const [text, setText] = useState('')
+  const suffix = `
```

Comment:
should we drop line or just give 1 space ?

Reply:
using a space or a blank line will occupy the same `.length`, I put it on the bottom line because I think it would be more readable, but I'll change it to inline (it doesn't matter much haha)